net-mgmt/zabbix-agent/src/opnsense/scripts/OPNsense/ZabbixAgent/setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+AGENT_DIRS="/var/run/zabbix /var/log/zabbix /usr/local/etc/zabbix_agentd.d"
```

Comment:
have you double checked with pkg-plist files? they can have other important dirs.

note this is also mostly for /var if MFS was enabled as /usr/local is persistent and likely created by the package install

https://github.com/opnsense/ports/tree/47aa039c8c8f361193e18a636760392971ef57ae/net-mgmt/zabbix24-server